airflow/models.py

```diff
@@ -2587,6 +2587,9 @@ def prepare_template(self):
         pass
 
     def resolve_template_files(self):
+        # Cast the template_fields to a list if it is defined as ('field')
+        if type(self.template_fields) == str:
```

Comment:
These should probably both be:

`if isinstance(self.template_fields, basestring):` to better support multiple python versions. This will need a `from past.builtins import basestring` too

Reply:
thanks for the info. Just updated.